src/preact/utils.js

```diff
@@ -16,15 +16,31 @@ export function useResourcesNotify() {
 }
 
 /**
- * Combines multiple refs to pass into `ref` prop.
- * @param {...any} refs
- * @return {function(!Element)}
+ * @param {any} ref
+ * @param {!Element} value
  */
-export function refs(...refs) {
-  return (element) => {
-    for (let i = 0; i < refs.length; i++) {
-      const ref = refs[i];
-      typeof ref == 'function' ? ref(element) : (ref.current = element);
+function setRef(ref, value) {
+  if (typeof ref === 'function') {
+    ref(value);
+  } else if (ref) {
+    ref.current = value;
+  }
+}
+
+/**
+ * Combines refs to pass into `ref` prop.
+ * @param {any} refOne
+ * @param {any} refTwo
+ * @return {function()}
+ */
+export function useMergeRefs(refOne, refTwo) {
```

Comment:
Through https://github.com/theKashey/use-callback-ref#usemergerefs I came across react-hooks.org. This is inspired from their `useForkRef` implementation. Tradeoff is it only combines 2 refs, but for now it does what is required.